src/app/register/register.component.html

```diff
@@ -0,0 +1,82 @@
+<h3 i18n>Become a member</h3>
+<form [formGroup]="registerForm" (ngSubmit)="registerUser(registerForm.value)">
+  <mat-grid-list cols="3">
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="First Name" formControlName="firstName" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.firstName"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="Middle Name" formControlName="middleName" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.middleName"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="Last Name" formControlName="lastName" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.LastName"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+  </mat-grid-list>
+  <mat-grid-list cols="3">
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="Login" formControlName="login" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.login"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input type="password" matInput i18n-placeholder placeholder="Password" formControlName="password" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.password"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input  type="password" matInput i18n-placeholder placeholder="Re-enter password" formControlName="repeatPassword" required>
+      <mat-error><planet-form-error-messages [control]="registerForm.controls.repeatPassword"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+  </mat-grid-list>
+  <mat-grid-list cols="3">
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="Email" formControlName="email" required>
```

Comment:
the input field email should only take valid email example : example@example.com
Please check  https://material.angular.io/components/input/examples for reference